scripts/linters/python_linter.py

```diff
@@ -45,158 +43,6 @@
 import pycodestyle # isort:skip  pylint: disable=wrong-import-order, wrong-import-position
 
 
-class PythonLintChecksManager(python_utils.OBJECT):
-    """Manages all the Python linting functions."""
-
-    def __init__(self, files_to_lint, file_cache):
-        """Constructs a PythonLintChecksManager object.
-
-        Args:
-            files_to_lint: list(str). A list of filepaths to lint.
-            file_cache: object(FileCache). Provides thread-safe access to cached
-                file content.
-        """
-        self.files_to_lint = files_to_lint
-        self.file_cache = file_cache
-
-    @property
-    def py_filepaths(self):
-        """Return all Python file paths."""
-        return self.files_to_lint
-
-    @property
-    def all_filepaths(self):
-        """Return all filepaths."""
-        return self.py_filepaths
-
-    def check_that_all_jobs_are_listed_in_the_job_registry_file(self):
```

Comment:
Nothing, since jobs are no longer registered using this system. The new system (using meta classes to keep track of registered classes), is being fine-tuned in the jobs directory.